tests/containers/apparmor.pm

```diff
@@ -0,0 +1,27 @@
+# SUSE's openQA tests
+#
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Package: apparmor-utils
+# Summary: Test apparmor utilities
+# Maintainer: QE-C team <qa-c@suse.de>
+
+use Mojo::Base 'containers::basetest';
+use testapi;
+use services::apparmor;
+use utils;
+
+sub is_enabled {
```

Comment:
We no longer need this subroutine since we're making sure to load this module only on systems where we expect AppArmor to be enabled.

Reply:
Indeed! Fixed in [786035](https://github.com/os-autoinst/os-autoinst-distri-opensuse/commit/786035acd76001ef788a432c17eaf5c98c8ff70d). I figured some redundancy wouldn't hurt, but it's cleaner this way. Also removed the call to systemctl since that's already covered in services::apparmor::check_function call.